backend/health-expense-calculator/src/main/resources/db/migrate.sh

```
@@ -1,3 +1,7 @@
 #!/bin/sh
+set -e
 
-flyway -url=$DB_URL -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS -baselineOnMigrate=true   -outOfOrder=true migrate
\ No newline at end of file
+FLYWAY_FLAGS="-url=$DB_URL -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS"
+
+flyway $FLYWAY_FLAGS repair
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)